libp2p/crypto/minasn1.nim

```diff
@@ -355,7 +356,7 @@ proc asn1EncodeOid*(dest: var openArray[byte], value: openArray[int]): int =
   var res = 1
   var oidlen = 1
   for i in 2..<len(value):
-    oidlen += asn1EncodeTag(buffer, cast[uint64](value[i]))
+    oidlen += asn1EncodeTag(buffer, cast[uint64](value[i])) # FIXME int to uint64 potentially unsafe
```

Comment:
It seems like this proc could take a `value: openArray[byte]` and it work without the cast

Reply:
There's a `proc asn1EncodeOid*(dest: var openArray[byte], value: openArray[byte]): int` defined right below. There's nothing using the one with int, I'm removing it.